lib/structures/Message.js

```diff
@@ -244,6 +249,14 @@ class Message extends Base {
             case MessageTypes.AUTO_MODERATION_ACTION: {
                 break;
             }
+            case MessageTypes.ROLE_SUBSCRIPTION_PURCHASE: {
+                data.content = `${this.author.mention + (this.roleSubscriptionData.is_renewal ? " renewed" : " joined")} **${this.roleSubscriptionData.tier_name}** ${!this.roleSubscriptionData.total_months_subscribed ? (this.roleSubscriptionData.is_renewal ?
```

Comment:
```suggestion
                data.content = `${this.author.mention} ${this.roleSubscriptionData.is_renewal ? "renewed" : "joined"} **${this.roleSubscriptionData.tier_name}** ${!this.roleSubscriptionData.total_months_subscribed ? (this.roleSubscriptionData.is_renewal ?
```
much simpler and concise :)